### DISCUSSION TITLE
Terragrunt - Github actions

### DISCUSSION THREAD

**Discussion by bprasad701** [gruntwork-io/knowledge-base#665] [How do I]
We are trying to  deploy infrastructure on to AWS using Terragrunt and Github actions.
when we run the github actions workflow script , it is showing "No changes Infrastructure is upto date" ,
but if we try to deploy infrastructure without using github actions we are able to that.

could someone please help me .

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/97897493/219068694-449efe86-b277-40f9-8cee-8be3de5aa90e.png)

---

<ins datetime="2023-02-15T15:15:16Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109902">Tracked in ticket #109902</a></p>
</ins>

**Comment by ellisonc** [comment]
@bprasad701 Could you provide some more information so we can help you debug? It would be helpful to see your github action configuration file. Are you certain your local deployment is pointed at the same AWS account the github action is working with?
